src/services/tx/tx-sender/sdk.ts

```diff
@@ -134,30 +133,18 @@ export const getUncheckedSigner = async (provider: Eip1193Provider) => {
  * dealing with smart-contract wallet owners
  */
 export const getUncheckedSafeSDK = async (provider: Eip1193Provider): Promise<Safe> => {
+  const sdk = getAndValidateSafeSDK()
   const browserProvider = createWeb3(provider)
   const signer = await browserProvider.getSigner()
   const uncheckedJsonRpcSigner = new UncheckedJsonRpcSigner(signer.provider, await signer.getAddress())
-  const sdk = getAndValidateSafeSDK()
-
-  const ethAdapter = new EthersAdapter({
-    ethers,
-    signerOrProvider: uncheckedJsonRpcSigner,
-  })
 
-  return sdk.connect({ ethAdapter })
+  return sdk.connect({ provider, signer: uncheckedJsonRpcSigner.address })
```

Comment:
We need to test if this actually does anything now since we are not passing the unchecked signer but just the signer address. Maybe there is a way to pass an "unchecked provider".

An easy way to test this is to execute a tx with a nested safe. The Tx modal should close immediately.